project.clj

```diff
@@ -1,8 +1,7 @@
-(defproject google-civic "2.0.2-SNAPSHOT"
+(defproject google-civic "2.1.0-SNAPSHOT"
   :description "Google Civic Info API client"
   :dependencies [[org.clojure/clojure "1.7.0"]
-                 [clj-http "2.0.0"]
-                 [cheshire "5.5.0"]
-                 [slingshot "0.12.2"]]
+                 [clj-http "2.3.0"]
```

Comment:
FWIW, we're using clj-http 3.x.x in at least the following projects already w/ no apparent issues:

1. ballot-scout
1. cuttlefish
1. election-authority-http-api
1. election-http-api
1. notification-works
1. turbovote-web
1. clj-smartystreets
1. mail-works
1. address-works
1. turbovote-admin-http-api
1. election-mail-http-api